src/PhpOption/LazyOption.php

```diff
@@ -88,9 +88,9 @@ public function getOrThrow(\Exception $ex)
         return $this->option()->getOrThrow($ex);
     }
 
-    public function orElse(Option $else)
+    public function orElse($else, $noneValue = null)
```

Comment:
Let's not add noneValue here. If you want to use something else, but ``null`` you just have to pass an Option.